src/brand.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Dart Brand Guidelines
+---
+
+The "Dart" name and logo are trademarks owned by Google.
+These Brand Guidelines describe the appropriate uses of the Dart
+trademarks by members of the developer community who have obtained our
+consent to use the trademarks pursuant to the [Dart Terms of Service](/terms).
+These guidelines will ensure that the Dart trademarks are used in a
+manner that promotes Google's mission to provide a free and open source
+SDK for crafting high-quality native interfaces on iOS and
+Android in record time, and are not associated with objectionable material,
+as determined by Google.  
+
+Use of the Dart trademarks that is not expressly permitted by these
+guidelines is prohibited absent written permission from Google.
+
+The official Dart logos can be found in the [Dart and Flutter logo
+assets](https://drive.google.com/corp/drive/folders/1KXNtO9My2AMpDOF9A9Y_4aj4_BcgmDDT) Drive folder.
+
+## General Rules That Govern the Use of the Dart Trademarks
```

Comment:
We use sentence case capitalization for headings:

## General rules that govern the use of the Dart trademarks